rules/eslint/es6/walmart.js

```diff
@@ -19,6 +27,12 @@ module.exports = {
     // require method and property shorthand syntax for object literals
     "object-shorthand": 2,
     // suggest using of const declaration for variables that are never modified after declared
-    "prefer-const": 2
+    "prefer-const": 2,
+    // suggest using the spread operator instead of .apply()
+    "prefer-spread": 0,
+    // suggest using Reflect methods where applicable
+    "prefer-reflect": 0,
+    // disallow generator functions that do not have yield
+    "require-yield": 0
```

Comment:
This should probably get moved to "Possible Errors" since any generator function without a yield won't do much. I say this one goes on.